pylintrc

```diff
@@ -173,7 +173,8 @@ disable=print-statement,
 # it should appear only once). See also the "--disable" option for examples.
 enable=c-extension-no-member,
        line-too-long,
-       use-a-generator
+       use-a-generator,
+       useless-object-inheritance
```

Comment:
I think this change will conflict with https://github.com/mlflow/mlflow/pull/6321. Can you resolve conflicts once https://github.com/mlflow/mlflow/pull/6321 is merged?

Reply:
Of course.